server/src/project/_utils/transform-milestones.ts

```diff
@@ -146,6 +146,21 @@ function applyDisplayDate(milestone, project) {
   if (milestone._dcp_milestone_value === '780593bb-ecc2-e811-8156-1458d04d0698') milestone.display_date = milestone.dcp_actualenddate;
   if (milestone._dcp_milestone_value === '483beec4-dad0-e711-8116-1458d04e2fb8') milestone.display_date = milestone.dcp_actualenddate;
   if (milestone._dcp_milestone_value === '4a3beec4-dad0-e711-8116-1458d04e2fb8') milestone.display_date = milestone.dcp_actualenddate;
+  
+  /*
+  * The following is for task 13239
+  * The date displayed should be the CRM Review Meeting date for certain milestones if the milestone is "in progress" or "completed", and the meeting date is <= current date
```

Comment:
would be clearer if you wrote "if the milestone _statuscode_ is "in progress" or "completed"

Reply:
I've updated the comment to clarify.

Reply:
Thanks!